climt/_core/initialization.py

```diff
@@ -787,7 +799,7 @@ def init_ozone(p):
 init_diagnostics = [
     PressureFunctionDiagnosticComponent(
         'longwave_optical_depth',
-        lambda p: 1. * (1. - p),
+        lambda p, ps: 1. * (1. - p/ps),
```

Comment:
After making the above changes to dimensionality, use `ps[None, :]`.

Reply:
I did not need to change the dimensionality, seems to work ok!